Header header
geometry_msgs/Quaternion q
geometry_msgs/Point nStar
float64 alpha
homography_vsc_cl/ImagePoints newPixels
float64 zStar
